jazzmin/templates/admin/change_form.html

```diff
@@ -86,16 +83,26 @@
             </div>
         {% endblock %}
 
-
         {% block after_field_sets %}{% endblock %}
 
         {% block inline_field_sets %}{% endblock %}
 
         {% block after_related_objects %}{% endblock %}
 
         {% block submit_buttons_bottom %}
-        <div class="col-4">
+        <div class="col-2">
```

Comment:
I think this is too short.. Do you want to use smaller buttons? TBH, it crossed my mind a couple of times as smaller looks also a bit jazzier. Big buttons are manky 😄